afids_utils/afids.py

```diff
@@ -54,6 +54,43 @@ class AfidPosition:
     z: float = attrs.field()
     desc: str = attrs.field(validator=_validate_desc)
 
+    def __sub__(self, other: AfidPosition) -> tuple[float, float, float]:
+        # Compute distances
+        x_dist = self.x - other.x
+        y_dist = self.y - other.y
+        z_dist = self.z - other.z
+
+        return x_dist, y_dist, z_dist
+
+
+@attrs.define(kw_only=True)
+class AfidDistance(AfidPosition):
```

Comment:
The semantics of this imply that it is specifically the distance between two AFIDs that are meant to identify the same anatomical feature (in the same image). In that case, I think it would be nice for it to maintain a reference to those two `AfidPosition`s (in which case it probably doesn't need to actually have any other attributes, and could be reconsidered as an `AfidPositionPair` or something). I'm not sure that actually makes sense though but would like to hear your thoughts

Reply:
If I understand, then I do think it makes sense to maintain a reference to the two `AfidPosition`s that are used to compute this, especially given that this a distance will want to be computed for each pair in the set.